application/bettle/bettle-entity/build.gradle

```diff
@@ -0,0 +1,9 @@
+apply plugin: "java"
```

Comment:
@ranjeet1515 : Add lombok plugin and reduce the code of setter and getter to annotation. 

Refer: https://projectlombok.org/mavenrepo/